pkgs/development/python-modules/llm-ollama/default.nix

```diff
@@ -41,16 +42,13 @@ buildPythonPackage rec {
     pydantic
   ];
 
+  preCheck = ''
+    export HOME="$(mktemp -d)"
+  '';
```

Comment:
🎩 to @GaetanLepage who just mentioned you can use `writableTmpDirAsHomeHook` by adding it to `nativeCheckInputs` over on #390715.

Reply:
Thank you both for teaching me this, PR updated!